script.js

```diff
@@ -241,4 +245,31 @@ form.addEventListener('submit', (e) => {
     email.style.border = '2px solid #df0000';
     email.style.borderRadius = '3px';
   }
-});
\ No newline at end of file
+});
+
+// Store form data in localStorage
+let formData = { name: '', email: '', message: '' };
+
+const saveToLocalStorage = () => {
+  localStorage.setItem('form_data', JSON.stringify(formData));
+};
+
+userName.addEventListener('change', () => {
+  formData.name = userName.value;
+  saveToLocalStorage();
+});
+email.addEventListener('change', () => {
+  formData.email = email.value;
+  saveToLocalStorage();
+});
+message.addEventListener('change', () => {
+  formData.message = message.value;
+  saveToLocalStorage();
+});
+
+window.onload = () => {
+  formData = JSON.parse(localStorage.getItem('form_data'));
+  userName.value = formData.name;
+  email.value = formData.email;
+  message.value = formData.message;
+};
```

Comment:
- [ ] When I tested your app locally I got this error : 
![image](https://user-images.githubusercontent.com/43698511/131905979-7bdccca5-5ddd-41b9-9b29-dceab1a7c082.png)
I think it's because you didn't check the case when the `'form_data'`  key doesn't exist in the locale Storage. In that case, you should set it first before trying to retrieve it, or else the formData object will be null.

Kindly fix the issue. You can reproduce the error by deleting all the values stored in the local storage before testing.